ops/jujuversion.py

```diff
@@ -34,7 +34,7 @@ def __eq__(self, other):
         if isinstance(other, str):
             other = type(self)(other)
         elif not isinstance(other, JujuVersion):
-            raise RuntimeError(f"Cannot compare Juju version {self} with {other}")
+            raise RuntimeError(f"cannot compare Juju version {self} with {other}")
```

Comment:
just thought of this. if we are quoting "{version}" up above, shouldn't we do that here?